Mining.md

```diff
@@ -104,29 +104,11 @@ The **minimum** requirements for Zilliqa mining nodes are:
 
 > **NOTE:** If you are using a home router, you are most probably in a NAT environment.
 
-If you are in NAT environment, you can either:
-
-- Do single port forwarding using **Option 1a**. This should be your **DEFAULT OPTION**.
-- Enable UPnP mode using **Option 1b** if your router does support UPnP.
-
 If you have a public IP address, you can skip this network setup entirely.
```

Comment:
We can say here like instead
"It is mandatory to have a public IP address. NAT is not supported."